pkg/sdn/plugin/poddb.go

```diff
@@ -0,0 +1,46 @@
+package plugin
```

Comment:
Yeah... I don't really like this. We should just pass the containerRuntime to the node so it can look up namespaces when it needs to. Yes, we'll need to do something else when we split the SDN out of the node process, but there are lots of other things we'll need to do something about as well, and this is no worse.

(The sync-all-pod-network-states-on-startup loop probably belongs in upstream kubernetes anyway; if that happened then we'd get an UPDATE-with-netns-filled-in for every pod at startup and then we'd be guaranteed to have a netns for every pod after that.)